src/scss/custom.scss

```diff
@@ -59,10 +62,123 @@
   }
 }
 
+// Side Bar
+.sidebar-section {
+  color: var(--doc-sidebar-section-title);
+  padding: var(--ifm-menu-link-padding-vertical) var(--ifm-menu-link-padding-horizontal);
+  text-transform: uppercase;
+  font-size: 14px;
+  font-weight: 600;
+}
+
+.sidebar-section:not(:first-child) {
+  margin-top: calc(var(--ifm-menu-link-padding-vertical) * 2);
+}
+
+.menu__list {
+  .menu__list {
+    padding-left: 0;
```

Comment:
or use negative margin? don't know if the hover state looks good

Reply:
Check its structure, feels the same but updated.